rspace/src/test/scala/coop/rchain/rspace/nextgenrspace/StorageActionsTests.scala

```diff
@@ -927,6 +928,58 @@ trait StorageActionsTests[F[_]]
       insertActions <- store.changes().map(collectActions[InsertAction])
     } yield (insertActions shouldBe empty)
   }
+
+  "createSoftCheckpoint" should "capture the current state of the store" in fixture {
```

Comment:
would a flow with two soft checkpoints make sense? something to show that they do not affect each others state